### PR TITLE
Instructor: add comment: fix bug that shows different visibility options when editing a newly added comment #7578

### LINKED ISSUE #7578: Instructor: add comment: fix bug that shows different visibility options when editing a newly added comment
problem
This can be seen as a bug by the instructors as changing the visibility option other than defined by the question have no effect on comment's visibility.

### PR DESCRIPTION
**Pull request by Mynk96**
Fixes #7578

### CODE REVIEW

**Comment by Mynk96** [comment]
UI Changes:-

solution

**Comment by Mynk96** [comment]
Ready For Review!

**Comment by Mynk96** [comment]
@leeyimin Are you reviewing this?

File: src/main/java/teammates/ui/pagedata/InstructorFeedbackResponseCommentAjaxPageData.java [hunk 1]
```diff
@@ -56,4 +59,31 @@ public FeedbackResponseCommentRow getComment() {
...
+                    && question.recipientType != FeedbackParticipantType.SELF
+                    && question.recipientType != FeedbackParticipantType.NONE
+                    && question.isResponseVisibleTo(FeedbackParticipantType.RECEIVER_TEAM_MEMBERS);
+        case STUDENTS:
+            return question.isResponseVisibleTo(FeedbackParticipantType.STUDENTS);
+        default:
+            Assumption.fail("Invalid participant type");
+            return false;
+        }
+    }
```
Comment (chowyb) [inline src/main/java/teammates/ui/pagedata/InstructorFeedbackResponseCommentAjaxPageData.java]: Doesn't seem to be a way to use inheritance for this unfortunately, but I guess this further highlights the need to handle the TODO in `InstructorFeedbackResultsPageData`

**Pull request merged**